builtin/providers/aws/resource_aws_instance.go

```diff
@@ -127,40 +130,56 @@ func resourceAwsInstance() *schema.Resource {
 				ForceNew: true,
 				Optional: true,
 			},
+
 			"tenancy": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
 				ForceNew: true,
 			},
+
 			"tags": tagsSchema(),
 
 			"block_device": &schema.Schema{
+				Type:     schema.TypeMap,
+				Optional: true,
+				Removed:  "Split out into three sub-types; see Changelog and Docs",
+			},
```

Comment:
First use of `Removed` - intended UX here for user that's upgraded:

 1. Receive this message
 2. Update config
 3. Catch state migration on next plan or apply which, if all goes well, should result in an empty plan